transaction.proto

```diff
@@ -15,12 +16,14 @@ message Fee {
     JubJubCompressed R = 3;
     JubJubCompressed pk_r = 4;
```

Comment:
Remember that we don't need this: every time you have the couple `R` + `pk_r` means you basically have a `StealthAddress` type. Let's sure we're consistent in that across our schemas.

Reply:
Good point - I already caught on to this doing some work for the rusk transaction module, but it makes sense to update it here too.